hollow/src/main/java/com/netflix/hollow/core/memory/ByteArrayOrdinalMap.java

```diff
@@ -433,7 +433,7 @@ private void growKeyArray() {
        populateNewHashArray(newKeys, valuesToAdd);
 
        /// 70% load factor
-       sizeBeforeGrow = (newKeys.length() * 7) / 10;
+       sizeBeforeGrow = (int) (((float) newKeys.length()) * 0.7);
```

Comment:
Might also be a good idea to throw an IllegalStateException (or something) with a useful error message if we overflow

Reply:
Yea, good idea. I've added the changes to throw an `IllegalStateException` when the new size overflows. This will help provide more clarity on the error.